core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -14,7 +14,7 @@
 
     <style>
       html, body {
-        background: url('/images/general/background.jpg') no-repeat center center fixed;
+        background: url("{{get_static_resource_url('/assets/images/general/background.jpg')}}") no-repeat center center fixed;
```

Comment:
We are trying to get away from using Jinja, so introducing Jinja in more places (e.g. CSS) is somewhat counterproductive.

Can we convert to something like this perhaps? http://stackoverflow.com/questions/13781685/angularjs-ng-src-equivalent-for-background-imageurl